honeycombio/type_helpers.go

```diff
@@ -97,8 +97,16 @@ func sortOrderStrings() []string {
 	return out
 }
 
-func triggerRecipientTypeStrings() []string {
-	in := honeycombio.TriggerRecipientTypes()
+func recipientTypeStrings(alertType string) []string {
```

Comment:
Using a string param like this is kind of icky. Might be better to pass in a const or maybe `honeycombio.TriggerRecipientTypes()` directly. Alternatively, could define `type RecipientTypes []RecipientType` and put something like a `AsString() []string` method on it so this helper can be directly on the type.